typed-env/src/main/scala/uscala/util/Env.scala

```diff
@@ -10,7 +10,7 @@ object Env {
   type Env = Map[String, String]
 
   /** The default map of environment variables will come from `System.getenv` */
-  val DefaultEnv: Env = JMapWrapper(System.getenv).toMap
+  val DefaultEnv: Env = System.getenv.asScala.toMap
```

Comment:
Using `JMapWrapper` is unsafe, it's intended to be an internal class and is gone in Scala 2.13. We switch here to using the safe and promoted way, the implicit `JavaConverters`.